gatsby-node.js

```diff
@@ -1,7 +1,61 @@
-/**
- * Implement Gatsby's Node APIs in this file.
- *
- * See: https://www.gatsbyjs.org/docs/node-apis/
- */
+const path = require('path');
 
```

Comment:
If using `createFilePath` then would need to bring that in as well
```suggestion
const { createFilePath } = require(`gatsby-source-filesystem`);
```